examples/basic-event-handlers/contracts/Gravity.sol

```diff
@@ -0,0 +1,62 @@
+pragma solidity ^0.4.0;
```

Comment:
This is pretty old solidity. Same with the truffle packages used. 

I don't believe it really matters. AFAIK the solidity compiler is backwards compatible and shouldn't effect our testing at all. But I figured I would point it out, in case you think it could effect us

Reply:
Doesn't matter much as long as these contracts compile and deploy fine to Ganache. It's just an example subgraph for testing.